api/types/filters/parse.go

```diff
@@ -211,20 +210,10 @@ func (args Args) GetBoolOrDefault(key string, defaultValue bool) (bool, error) {
 
 	isFalse := fieldValues["0"] || fieldValues["false"]
 	isTrue := fieldValues["1"] || fieldValues["true"]
-
-	conflicting := isFalse && isTrue
-	invalid := !isFalse && !isTrue
-
-	if conflicting || invalid {
+	if isFalse == isTrue {
```

Comment:
Right, but that was the case before as well, correct? 

```
conflicting := isFalse && isTrue
invalid := !isFalse && !isTrue
if conflicting || invalid {
```

The above; if _both_ are false or _both_ are true we return default value (with an error), so that's the same as `if `isFalse == isTrue`

Reply:
Oh, right.
I guess that's why I went with a more explicit code, so it's easier to wrap your head around truthness and falseness of `isFalse` and `isTrue` variables 😅

Reply:
I can add a comment "a boolean cannot be both true and false" ?

Reply:
Not sure that's helpful 🙈 

Maybe something like:
```suggestion
    // Either no or conflicting truthy/falsy value were provided 
	if isFalse == isTrue {
```

Reply:
@vvoland updated; added comment; PTAL